akka-sample-persistence-dc-scala/README.md

```diff
@@ -6,6 +6,8 @@ which is available as a commercial feature as part of the [Lightbend Subscriptio
 
 ## How to run
 
+You will need to have the `protobuf` compiler installed to convert `proto` definitions to object code.
```

Comment:
Any specific version of protobuf?

Reply:
Not that I know specifically.. 'the default' both on my laptop and on ubuntu was OK